packages/logger/index.js

```diff
@@ -0,0 +1,123 @@
+const logger = require('../dd-trace/src/log')
+// path to require tracer logger
+
+const https = require('https')
+const coalesce = require('koalas')
```

Comment:
I think `coalesce` is fine to keep using in the config files, but for application code, let's just use a pure language solution instead. Based on the usage I think it's safe to replace everything like so:

```js
before = coalesce(firstValue, secondValue)
after = firstValue || second Value
```

This should be slightly more performant, which can be important in cases where we generate tons of logs. It's also nice because a new developer probably won't know what the koalas package does but should understand the `useThis || fallback` pattern.